tests/golos.publication_tests.cpp

```diff
@@ -194,12 +215,23 @@ BOOST_FIXTURE_TEST_CASE(create_message, golos_publication_tester) try {
     BOOST_TEST_MESSAGE("Create message testing.");
     init();
     init_params();
+    init_referral();
+
     BOOST_CHECK_EQUAL(success(), post.create_msg(N(brucelee), "permlink"));
 
+    auto expire = 8; // sec
+    BOOST_CHECK_EQUAL(success(), referral.create_referral(N(dan.larimer), N(chucknorris), 500,
```

Comment:
I am opposed to testing a bunch of posting a contract and a referral program contract in basic posting contract tests.
Please, make this test in referral contract tests (as for social contract, see https://github.com/GolosChain/golos-smart/blob/master/tests/golos.social_tests.cpp#L109-L143).

And remove all changes from base tests.